.changeset/wild-pianos-check.md

```diff
@@ -0,0 +1,6 @@
+---
+'@capacitor-firebase/authentication': patch
+'@capacitor-firebase/app-check': patch
```

Comment:
```suggestion
```

Please remove this.

Reply:
<img width="1362" alt="image" src="https://github.com/user-attachments/assets/b7be5c29-b232-4004-9fa2-ccce01ad2bd8" />

`npm run fmt` command will cause formatting changes in `packages/app-check/ios/Plugin/FirebaseAppCheckPlugin.swift`. I am not sure whether it will block the building flow, so I committed it and marked a `@capacitor-firebase/app-check` patch version too.

I will remove it.

Reply:
It's okay to format the code in this package. But that's nothing that should be mentioned in the changelog. So just remove this line. 🙌